pyntc/devices/eos_device.py

```diff
@@ -46,6 +47,18 @@ def __init__(self, host, username, password, transport="http", timeout=60, **kwa
         self.timeout = timeout
         self.connection = eos_connect(transport, host=host, username=username, password=password, timeout=timeout)
         self.native = EOSNative(self.connection)
+        # _connected indicates Netmiko ssh connection
+        self._connected = False
+        # ftp corresponds to file transfer and openning a netmiko connection
+        if self.transport == "ftp":
+            self.open()
```

Comment:
should wait to call `self.open()` until file copy method is called

Reply:
We could do open by default, however when we do not transfer files we do not need to have a netmiko SSH connection. It seemed like a waste, but it does not change the behavior to get rid of the if statement.